src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -103,6 +103,8 @@ const enum Constants {
 	MaxCanvasWidth = 8000
 }
 
+const pinnedRecentCommandStorageKey = 'terminal.pinnedCommands';
```

Comment:
We have a central place for this:

https://github.com/microsoft/vscode/blob/36e2df6fcac304b1ffec440f53b3a62affccece1/src/vs/workbench/contrib/terminal/common/terminalStorageKeys.ts#L6